hw/ip/aes/doc/checklist.md

```diff
@@ -110,8 +110,8 @@ Review        | Signoff date            | Not Started |
 
  Type         | Item                                  | Resolution  | Note/Collaterals
 --------------|---------------------------------------|-------------|------------------
-Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | Done        | [AES DV Plan]({{<relref "hw/ip/aes/doc/dv_plan" >}})
-Documentation | [TESTPLAN_COMPLETED][]                | Done        | [AES Testplan]({{<relref "hw/ip/aes/doc/dv_plan/index.md#testplan" >}})
+Documentation | [DV_DOC_DRAFT_COMPLETED][]           | Done        | [AES DV Document]({{<relref "hw/ip/aes/doc/dv_doc" >}})
```

Comment:
Do you mind align the "|" before "Done" to match the rest of the lines?

Reply:
fixed